pkg/manager/member/ticdc_member_manager.go

```diff
@@ -127,6 +127,16 @@ func (m *ticdcMemberManager) Sync(tc *v1alpha1.TidbCluster) error {
 		return nil
 	}
 
+	ns := tc.GetNamespace()
+	tcName := tc.GetName()
+
+	if tc.Spec.PD != nil && !tc.PDIsAvailable() {
```

Comment:
If pd or tikv is not available, the status of cdc won't update, is it acceptable?

Reply:
Yes, TiCDC requires PD and TiKV service.

Reply:
Please note that with this change, all the operations, e.g. creation, scale, upgrade will be blocked. Is this expected?

Reply:
Thanks for reminder, I'll add some comments here.